### ISSUE TITLE
Unknown attribute kind (70)

### ISSUE THREAD

**Issue by oxinabox** [google-research/dex-lang#585]
New laptop (ubuntu this time), new failures.
Freshly built clone from current `main` errors as soon as I start the REPL.
```
$ stack exec -- dex repl                                                                17:56:47
Compiler bug!
Please report this at github.com/google-research/dex-lang/issues

ParseFailureException "Unknown attribute kind (70) (Producer: 'LLVM12.0.0' Reader: 'LLVM 9.0.1')"
```
I guess it is mixing LLVM versions somewhere.

**Comment by oxinabox** [comment]
Fixed by using Clang v11 rather than Clang v12

**Comment by apaszke** [comment]
We should do some stricter version checking in the makefile 😕

**Comment by axch** [comment]
Looks like this is fixed-enough by #639, so I'm going to close it.
